Cargo.toml

```diff
@@ -206,6 +206,11 @@ arrow-select = { version = "50" }
 parquet = { version = "50", features = ["async"] }
 parquet_rs = { package = "parquet", version = "50" }
 
+# Crates from risingwavelabs
+arrow-udf-js = { package = "arrow-udf-js", git = "https://github.com/risingwavelabs/arrow-udf", branch = "main" }
```

Comment:
please merge main and use the commit from extra lib.

Reply:
@sundy-li, I have merged the changes from the main branch of the upstream repository. Regarding the 'datafuse-extra' library, are there any specific instructions or considerations I should be aware of?"

Reply:
The main branch in "https://github.com/risingwavelabs/arrow-udf" had some private codes in risingwave https://github.com/risingwavelabs/arrow-udf/issues/16

So I forked a branch in :

arrow-udf-js = { package = "arrow-udf-js", git = "https://github.com/datafuse-extras/arrow-udf", rev = "0d1a13c" }

Reply:
The migration to the `datafuse-extras` forked version has been completed. Please review and suggest any necessary changes for the `rev` version.

```
# Crates from risingwavelabs
arrow-udf-js = { package = "arrow-udf-js", git = "https://github.com/datafuse-extras/arrow-udf", rev = "0d1a13c" }
arrow-udf-wasm = { package = "arrow-udf-wasm", git = "https://github.com/datafuse-extras/arrow-udf", rev = "0d1a13c" }
```